source/scss/components/_ibe-icons.scss

```diff
@@ -0,0 +1,15 @@
+.ibe-type-icons {
+  align-items: center;
+  display: inline-flex;
+  line-height: 26px;
+  vertical-align: middle;
+
+  i {
+    font-size: 26px;
+  }
+
+  > span {
+    font-size: 16px;
```

Comment:
Please use our typography-variables.scss under variables. Use our best friend font-size, or give it a more specific variable.

Reply:
That's right, thanks! I'll update it on the other places as well.